src/core/lib/security/credentials/iam/iam_credentials.cc

```diff
@@ -30,6 +28,10 @@
 #include <grpc/support/string_util.h>
 #include <grpc/support/sync.h>
 
+#include "absl/strings/str_format.h"
+#include "src/core/lib/gprpp/ref_counted_ptr.h"
```

Comment:
The include order of this file is weird. But I think you somehow had a merge issue to have duplicate includes.